client/me/account/main.jsx

```diff
@@ -491,15 +519,17 @@ const Account = React.createClass( {
 				<FormButton
 					isSubmitting={ this.state.submittingForm }
 					disabled={ ! this.props.userSettings.hasUnsavedSettings() || this.getDisabledState() || this.hasEmailValidationError() }
```

Comment:
For sure! I wasn't sure how far to go with refactoring - I'll quickly update this before merging :D